compiler/transforms/insert_accfg_op.py

```diff
@@ -31,9 +31,7 @@ def match_and_rewrite(self, op: builtin.ModuleOp, rewriter: PatternRewriter):
         assert t is not None
         acc_string = self.acc_op.name_prop.string_value()
         if t.lookup_symbol(op, acc_string) is not None:
-            raise RuntimeError(
-                f"Cannot insert op: @{acc_string}, already present in symbol table."
-            )
```

Comment:
Why are you removing this?

Reply:
In my test, I specify my own accfg op. I want to avoid having a different pipeline for every test, so I thought this would be preferenrtial behaviour: Insert the default accfg op, but if you want to override it with your own, then fine.

Reply:
ok nvm i put it back